index.html

```diff
@@ -71,12 +71,39 @@ <h2>I am a Software Developer</h2>
 		<div class="text1">
 			<h2>My Recent Works <img src="images/line.png" alt=""></h2>
 		</div>
+
+		<div class="popupcontainer" id="popup">
+			<div class="popupcontainer2">
+				<div class="popuptitle">
+					<h1 class="header2">Multi Post Story</h1>
+					<button class="closebtn active" type="button" onclick="togglePopup()">&times;</button>
+				</div>
+				<div class="langcontainer">
+					<ul class="ulcontainer">
+						<li class="langg">html</li>
+						<li class="langg">bootstrap</li>
+						<li class="langg">Ruby on rails</li>
+					</ul>
+				</div>
+				<div class="popupcontainer3">
```

Comment:
- [ ] The whole of this popup is required to be displayed dynamically. This means:
 - Using javascript to display the content
 - It is required to use the data for the project section for the popup too.

Thanks to Microverse for this, adding projects in the future will be easy for you. You will only add it in one place - as another object in the projects array.